### ISSUE TITLE
Website down

### ISSUE THREAD

**Issue by rightpad** [corollari/waspline-reader#10]
wasplinereader.com appears to be down

**Comment by san-r** [comment]
Yes. I came here to check for further development. It has not been updated for about two years. It still works fine on most sites though - so I'm not complaining.
